cmd/main.go

```diff
@@ -257,6 +261,45 @@ func main() {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
 
+	go func() {
+		kclient, err := mesherykube.New(nil)
+		if err != nil {
+			log.Error(err)
+		}
+		mesheryclient, err := operatorClient.New(&kclient.RestConfig)
```

Comment:
If `err == true` will `kclient == nil`? In that case this line may panic?

Reply:
ahh.. this was not inside a goroutine before, cuz I wanted it to be synchronous, that's why there are no return statements. But then I decided to put them in a goroutine but failed to add return statements 😅 . Thanks for pointing these @utkarsh-pro , will make the necessary changes.